web/src/app/modules/shared/pipes/ansiPipe/ansi.pipe.ts

```diff
@@ -15,9 +15,7 @@ import { default as AnsiUp } from 'ansi_up';
 export class AnsiPipe implements PipeTransform {
   ansiUp = new AnsiUp();
 
-  constructor(private sanitizer: DomSanitizer) {
-    this.ansiUp.escape_for_html = false;
```

Comment:
Were we depending on this behavior anywhere? I'm wondering why we were explicitly setting it to false.

Reply:
@wwitzel3 That is now enforced to be false to prevent security risks and therefore removed from the interface. So, the internal behavior is pretty much same as before. I also tested manually ansi codes work as before.